### PR TITLE
Use the new targz bundle format with the new PHP installer

### PR DESCRIPTION
**Pull request by labbati**
### Readiness checklist
- [x] (only for Members) Changelog has been added to the release document.
- [x] Tests added for this feature/bug.

### Reviewer checklist
- [ ] Appropriate labels assigned.
- [ ] Milestone is set.
- [ ] Changelog has been added to the release document. For community contributors the reviewer is in charge of this task.

### CODE REVIEW

File: dd-library-php-setup.php
Comment (labbati) [inline dd-library-php-setup.php:91]: `test-actions` is temporary for CI, will have to be changed right before the final approval to `dd-trace-php`

File: dockerfiles/verify_packages/installer/_test_upgrade_from_php_installer.sh
Comment (labbati) [inline dockerfiles/verify_packages/installer/_test_upgrade_from_php_installer.sh:16]: This will have to be changed to 0.68.0 right before merge (but tests will fail)

File: dockerfiles/verify_packages/installer/test_alpine_no_ext_json.sh
Comment (labbati) [inline dockerfiles/verify_packages/installer/test_alpine_no_ext_json.sh:13]: I tricked the system. This comes from https://github.com/labbati/test-actions/releases to run tests, then it will be changed to dd-trace-php before merging

**Review by morrisonlevi — DISMISSED** [review]
From the code perspective, everything looks good except for the URLs going into CircleCI.

It's not a blocker for this PR, but the error message in this case could be improved (probably just show the help message?):

```
$ php dd-library-php-setup.php 
ERROR: One and only one among --version, --url and --file must be provided
```

Is this installer expected to work on Mac? It looks like it's using linux-isms like `ldconfig` which don't work on Mac, obviously:

```
$ php dd-library-php-setup.php --version 0.68.0
sh: ldconfig: command not found
ERROR: Cannot find library 'libcurl'
Failed command (return code 1): ldconfig -p | grep libcurl
---- Output ----

---- End of output ----
```

When I install it in Linux, I see this at the end:
```
SUCCESS

Run this script in a non interactive mode adding the following 'php-bin' options:
  php 'dd-library-php-setup.php' '--version' '0.68.0' '--php-bin=php' '--php-bin=php-fpm'
```

I think this needs to say something like:
> To run this script in a non interactive mode, use the following 'php-bin' options:
What do you think?

Clicking approve, as none of the things I've mentioned block a merge, except possibly the CircleCI links (your call, be responsible).

**Comment by labbati** [comment]
>`ERROR: One and only one among --version, --url and --file must be provided`

Done https://github.com/DataDog/dd-trace-php/pull/1421/commits/b0332528b2f04c27a71812ca397c646e1d0749d5

> Is this installer expected to work on Mac?

No, and I wouldn't worry about it for now

> I think this needs to say something like...

I like your message more 👍🏻 . Done https://github.com/DataDog/dd-trace-php/pull/1421/commits/b1c7236e8adfee0fdffdf03a562553ed64453350

> except possibly the CircleCI links (your call, be responsible).

Replaced with actual urls. As a consequence installer tests will fail and I will manually test it once we have the release out.

Thanks @morrisonlevi for the review. As soon as I get the final approval I will merge it.

**Review by morrisonlevi — APPROVED** [review]

**Pull request merged**
